third-party/CMakeLists.txt

```diff
@@ -106,7 +134,11 @@ if(USE_BUNDLED_LUAROCKS)
 endif()
 
 if(USE_BUNDLED_JEMALLOC)
-  include(BuildJeMalloc)
+  if (WIN32)
+    message(STATUS "Building jemalloc in Windows is not supported (skipping)")
+  else()
+    include(BuildJeMalloc)
```

Comment:
Is jemalloc useful on windows? or is the allocator good enough? Should we consider https://github.com/shines77/jemalloc-win32 ?